zerver/tests/test_migrations.py

```diff
@@ -1,113 +0,0 @@
-# These are tests for Zulip's database migrations.  System documented at:
-#   https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html
-#
-# You can also read
-#   https://www.caktusgroup.com/blog/2016/02/02/writing-unit-tests-django-migrations/
-# to get a tutorial on the framework that inspired this feature.
-
-from django.db.migrations.state import StateApps
-
-from zerver.lib.test_classes import MigrationsTestCase
-from zerver.lib.test_helpers import use_db_models
-
-# Important note: These tests are very expensive, and details of
-# Django's database transaction model mean it does not super work to
-# have a lot of migrations tested in this file at once; so we usually
-# delete the old migration tests when adding a new one, so this file
-# always has a single migration test in it as an example.
```

Comment:
Shouldn't we leave this file, unless we're substituting a new migration test?

Reply:
Ah, I just got to the next commit.  You should squash this change into the subsequent commit.